vantage6-client/vantage6/client/subclients/dataframe.py

```diff
@@ -67,26 +67,33 @@ def list(self, session: int = None, display=False) -> dict:
     @post_filtering(iterable=False)
     def create(
         self,
-        database: str,
+        label: str,
         image: str,
+        method: str,
         input_: dict,
-        session: int = None,
+        session: int | None = None,
+        store_id: int | None = None,
```

Comment:
We never use `_id` postfixes in the client (?) can we change this to just `store`?
```suggestion
        store: int | None = None,
```